src/main/resources/i18n/Enums_fr.properties

```diff
@@ -173,9 +173,9 @@ public.EcosystemType.DesertsAndXericShrublands=Déserts et zones arbustives xér
 public.EcosystemType.FloodedGrasslandsAndSavannas=Prairies et savanes inondées
 public.EcosystemType.Mangroves=Mangroves
 public.EcosystemType.MediterraneanForestsWoodlandsAndScrubs=Forêts, bois et maquis méditerranéens
-public.EcosystemType.MontaneGrasslandsAndShrublands=Prairies et arbustes montagnards
+public.EcosystemType.MontaneGrasslandsAndShrublands=Prairies et arbustes de montagne
```

Comment:
You need to edit enum translations in Phrase. This change will be overwritten next time Phrase does a GitHub export.

Reply:
I made the changes in Phrase (https://github.com/terraware/terraware-server/pull/3091) and removed them from this PR